.env

```diff
@@ -63,6 +63,7 @@ ESPRESSO_SEQUENCER_PERMISSIONED_PROVER=0x14dc79964da2c08b23698b3d3cc7ca32193d995
 ESP_TOKEN_INITIAL_SUPPLY=3590000000
 ESP_TOKEN_NAME=Espresso
 ESP_TOKEN_SYMBOL=ESP
+ESPRESSO_SEQUENCER_BLOCKS_PER_EPOCH=10
```

Comment:
This is kind of confusing now. I first thought this should break the demos that do upgrades (those require at least 150 block per epoch) due to guardrails in hotshot upgrades. I think the reason why it works anyway is because during deployment we fetch `blocks_per_epoch` from the sequencer API instead.

Do we actually need this to be configurable via CLI argument / env var?

Reply:
that's fair, addressed here https://github.com/EspressoSystems/espresso-network/pull/3479/commits/5d84a89c1d29d2e5c23f596027622e948c5c7dc7